### ISSUE TITLE
feature request: Add history to selectors

### ISSUE THREAD

**Issue by fleytman** [appium/appium-inspector#1584]
### Current Behavior

Typical mechanism for using the appium inspector

1. Open a session or join a session
2. Click on the magnifying glass icon
3. Test your selector
4. Repeat in a another session

Problems:
1. At step 3, you may need to check the selector in id, xpath, etc. In this case, the last entered value is saved in the field, regardless of the tab.
2. At step 4, the field "Selector" will be empty

### Suggested Solution

1. Store the selector value separately for each "Locator Strategy" tab
2. Save multiple selector values, such as 100 (or manage the number in the settings)
3. Store the value between sessions

**Comment by eglitise** [comment]
Thanks, this feature absolutely makes sense. I think a reasonable approach would be to just have a new 'history' button in the search window, saving the last X strategies+selectors. This would be saved between sessions, and clicking any entry would populate the respective fields.

**Comment by fleytman** [comment]
The button with the history is quite normal - as the most obvious and simple one. If you are interested, what could be more convenient, but probably more complex?
1. Switch between options with the up/down arrows as in the terminal
2. Suggest for autocomplete with the ability to switch between suggestions

**Comment by itkhanz** [comment]
I also agree this feature would be pretty useful. Currently I have to write/copy locators again if I am to try any locators. Having them saved in history tab would allow users to quickly pick up the locator from recently tried locators history. This will be handy when we are trying to build complex XPath locators in absence of IDs in order to build a suitable locator.
